apps/site/src/pages/wordpress.page.tsx

```diff
@@ -0,0 +1,34 @@
+import { NextSeo } from "next-seo";
+import { FunctionComponent } from "react";
+
+import { Availability } from "../components/pages/wordpress/availability";
+import { EmailSubmittedProvider } from "../components/pages/wordpress/email-submitted-context";
+import { Header } from "../components/pages/wordpress/header";
+import { WhyDevelop } from "../components/pages/wordpress/why-develop";
+import { WhyInstall } from "../components/pages/wordpress/why-install";
+
+const WordPressPage: FunctionComponent = () => (
+  <>
+    <NextSeo
+      title="Block Protocol WordPress Plugin"
+      openGraph={{
+        title: "Þ plugin for WordPress",
```

Comment:
```suggestion
        title: "Block Protocol plugin for WordPress",
```

This looked worse than I thought in most preview environments 😆  Sorry